frontend/src/metabase/query_builder/components/filters/modals/BulkFilterList/utils.ts

```diff
@@ -2,7 +2,7 @@ import { DimensionOption } from "metabase-lib/lib/queries/StructuredQuery";
 
 import { isDate } from "metabase/lib/schema_metadata";
 
-const LONG_TEXT_MIN = 80;
+import { LONG_TEXT_MIN } from "metabase-lib/lib/metadata/Field";
```

Comment:
this made sense to me as a home for this constant, but not entirely sure if there's a more sensible place for it.